test.js

```diff
@@ -69,7 +72,12 @@ test('does it work?', function (t) {
             var token = res.result;
             t.ok(token);
             server.inject({ method: "GET", url: '/2?notice=' + token}, function (res) {
-                t.equal(res.result.trim(), 'notice: boom');
+                var renderedNotices = res.result.trim().split('\n').map(function(value) {
```

Comment:
As an outsider coming in I'd love to see the tests split into a few assertions, rather than everything being contained in the top-level description, `does it work?` perhaps we could have something like this:

```js
describe('hapi-stateless-notifications', function () {
  it('handles success notices', function () {

  })

  it('handles error notices', function () {

  })

  it('handles a mix of success and error notifications', function () {

  })
})
```

Getting this foundation in place will make it super easy for other folks to swoop in and add functionality \o/